libnmstate/validator.py

```diff
@@ -75,6 +75,14 @@ class NmstateOvsLagValueError(NmstateValueError):
 
 def validate(data, validation_schema=schema.ifaces_schema):
     data = copy.deepcopy(data)
+    num_of_interfaces = len(
+        [intface for intface in data.get(schema.Interface.KEY, ())]
+    )
+    supported_interface_limit = 1000
+    if num_of_interfaces > supported_interface_limit:
+        raise NmstateValueError(
```

Comment:
Please use `logging.warning` instead of raise exception.

Reply:
@cathay4t 
Fixed. Shall I update man page also

Reply:
That would be great.